.github/workflows/dagger-golangci-lint.yml

```diff
@@ -17,15 +17,15 @@ permissions:  # added using https://github.com/step-security/secure-repo
 jobs:
   golangci:
     permissions:
-      contents: read  # for actions/checkout to fetch code
+      contents: read  # for actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```

Comment:
Was this intentional? This is a comment, so I'm not sure if dependabot will keep this up-to-date as well 🤔

Reply:
No, this is a `sed` skill issue on my part. 😹

Reply:
Added a commit reverting it then :)